ios/samples/HelloCocoaPods/HelloCocoaPods/ViewController.m

```diff
@@ -0,0 +1,233 @@
+/*
+* Copyright 2020 The Android Open Source Project
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#import "ViewController.h"
+
+#import <MetalKit/MTKView.h>
+
+// These are all C++ headers, so make sure the type of this file is Objective-C++ source.
```

Comment:
Should this file have an `.mm` extension?

Reply:
Changing the extension to `.mm` didn't seem to trigger Xcode into recognizing it in Objective-C++, so I had to explicitly change the file type. Doesn't hurt to do both, I suppose.